CallCenterSample/Helpers/TextAnalyticsHelper.cs

```diff
@@ -109,48 +123,41 @@ public static async Task<DetectLanguageResult> GetDetectedLanguageAsync(string i
 
             if (!string.IsNullOrEmpty(input))
             {
-                LanguageBatchResult result = await AnalyticsClient.DetectLanguageAsync(new BatchInput(
-                    new List<Input>()
-                    {
-                        new Input("0", input)
-                    }));
-
-                if (result.Documents != null)
-                {
-                    languageResult.Language.Add("iso6391Name", result.Documents[0].DetectedLanguages[0].Iso6391Name);
-                    languageResult.Language.Add("name", result.Documents[0].DetectedLanguages[0].Name);
-                    languageResult.Language.Add("score", result.Documents[0].DetectedLanguages[0].Score.ToString());
-                }
+                DetectLanguageResultCollection result = await AnalyticsClient.DetectLanguageBatchAsync(
+                new List<string>() { input });
 
-                if (result.Errors != null)
+                if (!result[0].HasError)
                 {
-                    // Just return the empty Dictionary
-                }
+                    languageResult.Language.Add("iso6391Name", result[0].PrimaryLanguage.Iso6391Name);
+                    languageResult.Language.Add("name", result[0].PrimaryLanguage.Name);
+                    languageResult.Language.Add("score", result[0].PrimaryLanguage.ConfidenceScore.ToString());
+                }   
             }
 
             return languageResult;
         }
 
         public static async Task<SentimentResult> GetTextSentimentAsync(string input, string language = "en")
         {
-            SentimentResult sentimentResult = new SentimentResult() { Score = 0.5 };
+            SentimentResult sentimentResult = new SentimentResult()
```

Comment:
same question, why the need of a new type instead of returning `DocumentSentiment` directly?
See sample [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2_AnalyzeSentiment.md#analyzing-the-sentiment-of-a-single-document).

Reply:
I'm not sure will the properties of `DocumentSentiment` return be null on exceptions?
This way I can still return initial values representing empty/pointless result. This is also how it was before this PR.

Reply:
Yes the object will be null as nothing should be assigned to it.
I see. Do you think this will create misleading information?

Reply:
It also feels overly complicated. Because this is a sample, I worry that we are telling the user that they need to create new types because the data the SDK returns is not enough or good.
In principle, the sample should reflect what the SDK is capable of.